Lil-Peep/src/sections/featured-kits.liquid

```diff
@@ -188,6 +188,12 @@
          ],
          "default":   "1",
          "info":      "Just one of each type"
+       },
+       {
+         "type": "text",
```

Comment:
Let's use the collection component instead of a text component...
https://help.shopify.com/themes/development/theme-editor/settings-schema#collection
With that you are limiting the user to use only collections inside your store and it's going to be less error prone 🤓